log/logger.go

```diff
@@ -0,0 +1,68 @@
+package log
+
+import (
+	"github.com/op/go-logging"
+	"io"
+)
+
+// BaseLogger is a convenience interface that makes use of all functionality from go-logging's
+// Logger struct. In addition, it defines Disable() and SetLevel(level) functions
+type BaseLogger interface {
+	// These functions are our own
+	Disable()
```

Comment:
L12 is now removed in favor of `Logger`'s implementation `NullLogger`, whose sole function is to suppress logging output. This way loggers can just be replaced (for instance, when running test functions), instead of all having a `Disable()` function.